README.md

```diff
@@ -23,7 +23,7 @@ npm install -D remix-flat-routes
 import { remixConfigRoutes } from "@react-router/remix-config-routes-adapter";
```

Comment:
They renamed this from pre-release, it is now `@react-router/remix-routes-option-adapter`